db/mongo.js

```diff
@@ -0,0 +1,26 @@
+const { MongoClient } = require('mongodb');
+
+const dbName = 'abstgtplregistryva6';
```

Comment:
There's a db name variable here, but also a db name environment variable in `.env`, which one to use?